quarkchain/cluster/shard_state.py

```diff
@@ -1757,7 +1757,14 @@ def validate_minor_block_seal(self, block: MinorBlock):
             block.header.branch.get_full_shard_id()
         ].CONSENSUS_TYPE
         posw_diff = self.posw_diff_adjust(block)  # could be None
-        validate_seal(block.header, consensus_type, adjusted_diff=posw_diff)
+
+        with_rotation_stats = self._qkchashx_enabled(block.header)
```

Comment:
since qkchashx is specific to qkchash, how about:

```
validate_seal(
  ...,
  qkchash_with_rotation_stats=consensus_type == QKCHASH and self._qkchashx_enabled(block.header)
)
```